src/main/java/uk/gov/hmcts/reform/sandl/snlapi/config/PasswordEncoderConfig.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.sandl.snlapi.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class PasswordEncoderConfig {
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
```

Comment:
see https://www.owasp.org/index.php/Password_Storage_Cheat_Sheet, I believe better algorithm could be used here.

Reply:
See discussion here: https://coderanch.com/t/701942/frameworks/Password-encoding-Spring-Argon
I wanted to use something officially supported by Spring Security, and also taking into account where/how this is to be deployed

Reply:
I can change to scrypt easily, would that be OK?

Reply:
See here (link from SCryptPasswordEncoder in Spring Security):
https://blog.ircmaxell.com/2014/03/why-i-dont-recommend-scrypt.html
That is a few years old though...

Reply:
Changed to PBKDF2 on your advice.